build.gradle

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+/* Build Script */
+import org.gradle.internal.jvm.Jvm
+
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "gradle.plugin.com.palantir.gradle.docker:gradle-docker:0.22.1"
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:2.1.7.RELEASE"
+    }
+}
+
+plugins {
+    id "com.diffplug.gradle.spotless" version "3.24.1"
+    id "java"
+    id "maven"
+    id "org.owasp.dependencycheck" version "5.2.1"
+    id "org.sonarqube" version "2.7.1"
+}
+
+allprojects {
```

Comment:
✏️ We can move things out of the `allprojects` block. That's only need if you have submodules that you want to apply plugins et. all too.